docs/packages/working_with_packages.md

```diff
@@ -0,0 +1,326 @@
+# Working with Packages
+
+- [Image Config File](#image-config-file)
+    - [File Paths](#file-paths)
+    - [Package Lists](#package-lists)
+- [Customize the Demo Image with Pre-built Packages](#customize-the-demo-image-with-pre-built-packages)
+    - [Add Latest Pre-Built Package](#add-latest-pre-built-package)
+    - [Add Specific Pre-Built Package Version](#add-specific-pre-built-package-version)
+    - [Add Packages from Other RPM Repositories](#add-packages-from-other-rpm-repositories)
+- [Customize the Demo Image with New Packages](#customize-the-demo-image-with-new-packages)
+
+## Image Config File
+
+### File Paths
```

Comment:
Is this section necessarily? Or is it considered extra wording that we can remove?

Reply:
The filepaths part is important here, although maybe this could be reframed as something along the lines of "working with custom packages in your image".